reference/docs-conceptual/components/vscode/using-vscode.md

```diff
@@ -96,21 +97,22 @@ choose the version:
 1. Choose the version of PowerShell you want to use from the list - for example, "PowerShell Core".
 
 >[!IMPORTANT]
+
```

Comment:
This breaks rendering.  Also, line 99 above needs to be spaced properly for render.

Reply:
Fixed